src/legacy/core_plugins/state_session_storage_redirect/public/index.js

```diff
@@ -20,18 +20,22 @@
 import chrome from 'ui/chrome';
 import { hashUrl } from 'ui/state_management/state_hashing';
 import uiRoutes from 'ui/routes';
+import { fatalError } from 'ui/notify';
 
 uiRoutes.enable();
 uiRoutes
   .when('/', {
     resolve: {
       url: function (AppState, globalState, $window) {
         const redirectUrl = chrome.getInjected('redirectUrl');
+        try {
+          const hashedUrl = hashUrl(redirectUrl);
+          const url = chrome.addBasePath(hashedUrl);
 
-        const hashedUrl = hashUrl([new AppState(), globalState], redirectUrl);
-        const url = chrome.addBasePath(hashedUrl);
-
-        $window.location = url;
+          $window.location = url;
+        } catch (e) {
+          fatalError(e);
```

Comment:
Added to replicate behaviour which was handled inside stats

Reply:
Where was the original behavior in states? Having trouble finding what you are referring to

Reply:
The code flow ended up here:
https://github.com/elastic/kibana/blob/a35a6cac39594093b0d999aefb17c601b0c5d9b0/src/legacy/ui/public/state_management/state.js#L315